src/webui/www/private/views/search.html

```diff
@@ -9,21 +9,6 @@
         background-position: left;
     }
 
-    #categorySelect {
-        width: 150px;
-        height: 20px;
-    }
-
-    #pluginsSelect {
-        width: 150px;
-        height: 20px;
-    }
-
-    #startSearchButton {
-        width: 90px;
-        height: 20px;
-    }
-
```

Comment:
I don't believe this PR fixes #11553. You only need to remove `line-height: 2em;` from `#searchPattern`.

Reply:
>Without these static widths the UI jumps around as the selections change (e.g. selecting a plugin can change the width of the category select element). 

I'll revert/revise the width part. What I mean by revise is that the default width isn't sufficient for Chinese characters, the text look like it is not centered to the button, increasing the width fixes it for me.

>I don't believe this PR fixes #11553. You only need to remove line-height: 2em; from #searchPattern.

The fixed height is not sufficient for Chinese characters either. The characters are trimmed in the middle (near the bottom), so instead of giving a fixed height use the auto value. At least it works better for me.
ps. You can test it with the "正體中文" language. Maybe I can post a screenshot later when I find some time.

Reply:
Also I noticed the search categories (books, software, etc.) are not translated. Maybe you would like to fix that.

Reply:
> The fixed height is not sufficient for Chinese characters either.

I see what you mean now from your screenshot. 👍 

> Also I noticed the search categories (books, software, etc.) are not translated.

This issue has come up before in #10239 but I haven't had the time/motivation to fix it.